### PR TITLE
bittrex.has futuresMethods = false

### PR DESCRIPTION
**Pull request by samgermain**
There's no bittrex futures, so all the futures methods should be false

### CODE REVIEW

File: js/bittrex.js [hunk 1]
```diff
@@ -20,13 +20,16 @@ module.exports = class bittrex extends Exchange {
             'pro': true,
             // new metainfo interface
             'has': {
+                'spot': true,
                 'margin': false,
                 'swap': false,
                 'future': false,
+                'options': false,
```
Comment (kroitor) [inline js/bittrex.js:27]: Is it `options` or `option` ?

Comment (kroitor) [reply js/bittrex.js:27]: I think it should be `option` here.

File: js/bittrex.js [hunk 2]
```diff
@@ -307,6 +326,7 @@ module.exports = class bittrex extends Exchange {
                         'max': undefined,
                     },
                     'leverage': {
+                        'min': 1,
                         'max': 1,
                     },
```
Comment (kroitor) [inline js/bittrex.js:331]: I think, if they don't have `leverage`, then we should either remove `leverage` entirely, or set it to `undefined`, or set `min` and `max` to `undefined`.

File: js/bittrex.js (hunk 1 above)
Comment (samgermain) [reply js/bittrex.js:27]: option, sorry

File: js/bittrex.js (hunk 2 above)
Comment (samgermain) [reply js/bittrex.js:331]: https://github.com/ccxt/ccxt/pull/11302#discussion_r784654212

**Comment by samgermain** [comment]
merged into https://github.com/ccxt/ccxt/pull/11302

**Pull request closed without merging**